artcommon/tests/test_konflux_db.py

```
@@ -495,14 +635,42 @@ def test_generate_bundle_builds_schema(self):
+            ('rpm', ArtifactType.RPM),
+        ]
+
+        for string_val, expected_enum in test_cases:
+            if isinstance(expected_enum, KonfluxBuildOutcome):
+                result = KonfluxBuildOutcome(string_val)
+            elif isinstance(expected_enum, Engine):
+                result = Engine(string_val)
+            elif isinstance(expected_enum, ArtifactType):
+                result = ArtifactType(string_val)
+
+            self.assertEqual(result, expected_enum, f"String '{string_val}' should convert to {expected_enum}")
```

Pylint (4.0.2)

[error] Possibly using variable 'result' before assignment

(E0606)